package mvp;

message MVPUserSettings {
  required double alt_min = 1;
  required double alt_max = 2;
  required double alt_search_range = 3;
  required double seed_window_smooth_size = 4;
  required double window_size = 5;
  required double window_smooth_size = 6;
  optional double gauss_divisor = 7 [default=6];
  optional int32 max_iterations = 8 [default=80];
}
